### ISSUE TITLE
Build instructions

### ISSUE THREAD

**Issue by amouat** [ehazlett/docker-volume-libsecret#1]
I'm not sure if I'm being stupid, or the path to a dependency has changed, but I can't get this to build. I'm not a Go programmer, so I've been guessing. Just running the Makefile doesn't work. Using `go get` gets a bit further:

```

$ go get github.com/ehazlett/docker-volume-libsecret
package github.com/docker/docker/pkg/sockets
    imports github.com/docker/docker/pkg/sockets
    imports github.com/docker/docker/pkg/sockets: cannot find package "github.com/docker/docker/pkg/sockets" in any of:
    /usr/lib/go/src/pkg/github.com/docker/docker/pkg/sockets (from $GOROOT)
    /home/amouat/go/src/github.com/docker/docker/pkg/sockets (from $GOPATH)

```

Any suggestions? Also, any chance of providing a binary? I guess there are good security reasons not to trust a binary, but it would be a lot easier for testing.

**Comment by BenHall** [comment]
Hi Adrian,

I'm not sure what version of Go you're using. In my fork (https://github.com/BenHall/docker-volume-libsecret) I have a `make dev` command which launches a configured Go environment. From there you can run `go get` and `go build` to generate a binary. I've also added one to the release tab.

If you want to experiment, I've created a Katacoda scenario which launches Vault and LibSecret. Try it at https://www.katacoda.com/courses/docker-production/docker-volume-libsecret. 

I'd love to hear feedback on the scenario.

Ben

**Comment by amouat** [comment]
Thanks Ben. I'll try to take a look.

**Comment by amouat** [comment]
Yeah, seems to be a Go 1.5 vs 1.6 issue. Feel free to close, but you may want to add this to the docs so I'll leave it open for the minute.

**Comment by rkuzsma** [comment]
Just a shout-out to @BenHall ... Loved your Katacoda scenario on docker-volume-libsecret.

**Comment by mmukherjee** [comment]
So here is my crazy hack - clearly not the best thing to do.

I got the binary out of the docker image at - https://hub.docker.com/r/ehazlett/docker-volume-libsecret/
